protos/SignalService.proto

```diff
@@ -34,24 +33,17 @@ message Content {
   optional CallMessage    callMessage    = 3;
   optional NullMessage    nullMessage    = 4;
   optional ReceiptMessage receiptMessage = 5;
+  optional PreKeyBundleMessage preKeyBundleMessage = 101;
 }
 
 message PreKeyBundleMessage {
-  enum Type {
-    UNKNOWN               = 0;
-    FRIEND_REQUEST        = 1;
-    FRIEND_REQUEST_ACCEPT = 2;
-    RESET_SESSION         = 3;
-    RESET_SESSION_ACK     = 4;
-  }
   optional bytes identityKey      = 1;
   optional uint32 deviceId        = 2;
   optional uint32 preKeyId        = 3;
   optional uint32 signedKeyId     = 4;
   optional bytes preKey           = 5;
   optional bytes signedKey        = 6;
   optional bytes signature        = 7;
-  optional Type type              = 8;
```

Comment:
so the envelope type tells us why the prekey bundle is sent?
FRIEND_REQUEST => FRIEND_REQUEST
other => RESET_SESSION

Reply:
Where is the reset session flag you mentioned yesterday? Is it inside the DataMessage?
We can put a flag in the prekeybundlemessage if it makes it cleaner, it was just redundant without the session reset functionality

Reply:
It's in `DataMessage`.
Yeah sorry my previous comment was not super clear.
So it's all good, I agree with removing the type, as we don't do anything with it (we save the prekeys and continue the logic flow regardless of the reason we attached the prekeys)